tests/cases/types/structs_keyword.slint

```diff
@@ -12,6 +12,10 @@ match := Rectangle {
     property<bool> test: move.loop == "mod";
 }
 
+export struct Name {
+    delete: image
+}
```

Comment:
you can also try to add more stuff such as
```slint
struct struct {
   delete: image;
}

export global const {
   in-out property<struct> return;
}
```

Reply:
I added some in new commits. Although in case of for exaple 

```
 export global const{ 
    in-out property<unsigned> atomic_commit;
    in-out property<namespace> return;  
 }
```
there is 
 ```
running 1 test
test test_cpp_types_structs_keyword ... FAILED

failures:

---- test_cpp_types_structs_keyword stdout ----
/tmp/.tmpn7hGBG.cpp: In member function ‘const T& char_::global() const’:
/tmp/.tmpn7hGBG.cpp:293:41: error: ISO C++ forbids declaration of ‘type name’ with no type [-fpermissive]
  293 |     else if constexpr(std::is_same_v<T, const>) { return *m_globals.global_const.get(); }
      |                                         ^~~~~
thread 'test_cpp_types_structs_keyword' panicked at /home/qubi/Projects/slint/target/debug/build/test-driver-cpp-815659352fb91da7/out/test_functions.rs:3100:20:
called `Result::unwrap()` on an `Err` value: "C++ Compilation error (see stdout)"
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```
I'm not sure if this should be adrressed or not.